LBH.AdultSocialCare.Api/V1/UseCase/PackageStatusUseCases/GetStatusUseCase.cs

```diff
@@ -14,7 +14,7 @@ public GetStatusUseCase(IStatusGateway roleGateway)
         {
             _gateway = roleGateway;
         }
-        public async Task<StatusDomain> GetAsync(Guid statusId)
+        public async Task<StatusDomain> GetAsync(int statusId)
         {
             var statusEntity = await _gateway.GetAsync(statusId).ConfigureAwait(false);
             return StatusFactory.ToDomain(statusEntity);
```

Comment:
`ToDomain()` should be in the gateway. In use case we do  `[Domain].ToResponse()`. Getting domain from gateway affords some flexibility. Instead of using `Includes` for many tables and getting entire objects from DB, we can get specific fields from DB direct into domain and return that domain from Gateway